src/components/shared/modals/BannyAttachModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { t } from '@lingui/macro'
+import { Col, Modal, Row, Space } from 'antd'
+import { ThemeContext } from 'contexts/themeContext'
+import { useContext, useState } from 'react'
+
+type AttachableBanny = {
+  url: string
+  alt: string
+}
+
+const ATTACHABLE_BANNYS: AttachableBanny[] = [
+  { url: '/assets/banny_eth.png', alt: 'Blockchain Banny' },
+  { url: '/assets/banny_love.png', alt: 'Banny in love' },
```

Comment:
In case you haven't already, can you run these images through https://imageoptim.com/mac ?

Reply:
Doneski